src/Chainweb/Rosetta/RestAPI/Server.hs

```diff
@@ -56,16 +60,16 @@ rosettaServer ms = (const $ error "not yet implemented")
     :<|> flip mempoolTransactionH ms
     :<|> flip mempoolH ms
     -- Network --
-    :<|> (const $ error "not yet implemented")
+    :<|> flip networkListH v
     :<|> (const $ error "not yet implemented")
     :<|> (const $ error "not yet implemented")
 
 someRosettaServer
     :: ChainwebVersion
     -> [(ChainId, MempoolBackend a)]
     -> SomeServer
-someRosettaServer (FromSingChainwebVersion (SChainwebVersion :: Sing vT)) ms =
-    SomeServer (Proxy @(RosettaApi vT)) $ rosettaServer ms
+someRosettaServer v@(FromSingChainwebVersion (SChainwebVersion :: Sing vT)) ms =
+    SomeServer (Proxy @(RosettaApi vT)) $ rosettaServer ms v
```

Comment:
Is there a way to get the Chainweb Version from inside the `rosettaServer` function?

Reply:
You seem to have accomplished that? Was there more you were looking for?

Reply:
It was purely an optional thing to not to pass in yet another parameter. Was just wondering if there was a way to access the chainweb node version from the state.

Reply:
We pass is it in like this in the other servers, so no harm done.

Reply:
that works then. Resolving comment